omero/developers/scripts/index.rst

```diff
@@ -58,7 +58,7 @@ include:
 
 All of the included scripts and repositories can be downloaded following the
 instructions below in order to run the scripts locally (although some of them
-are intended as examples only - check the associated README).
+are intended as examples only — check the associated README).
```

Comment:
If you're using an em-dash, you can remove the spaces around it, by the way.

Reply:
Hah! I removed them and then put them back in because it looked odd given I'm not using two to replace parentheses